--- darktable-1.1.4/src/common/imageio_exr.hh~	2013-03-09 22:48:35.000000000 +0100
+++ darktable-1.1.4/src/common/imageio_exr.hh	2013-04-05 08:11:04.499777789 +0200
@@ -31,7 +31,7 @@
 #include <OpenEXR/ImfStandardAttributes.h>
 
 // this stores our exif data as a blob.
-namespace Imf
+namespace Imf_2_0
 {
 class Blob
 {
@@ -54,7 +54,7 @@
 };
 
 
-typedef Imf::TypedAttribute<Imf::Blob> BlobAttribute;
+typedef Imf_2_0::TypedAttribute<Imf_2_0::Blob> BlobAttribute;
 template <> const char *BlobAttribute::staticTypeName()
 {
   return "blob";
