diff -ur darktable-2.6.0/src/chart/CMakeLists.txt darktable-2.6.0-rpath/src/chart/CMakeLists.txt
--- darktable-2.6.0/src/chart/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/chart/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -26,7 +26,7 @@
     endif()
     set_target_properties(darktable-chart
                           PROPERTIES
-                          INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                          INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
 endif(WIN32)
 
 install(TARGETS darktable-chart DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT DTApplication)
diff -ur darktable-2.6.0/src/cli/CMakeLists.txt darktable-2.6.0-rpath/src/cli/CMakeLists.txt
--- darktable-2.6.0/src/cli/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/cli/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -17,7 +17,7 @@
     endif()
     set_target_properties(darktable-cli
                           PROPERTIES
-                          INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                          INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
 endif(WIN32)
 
 install(TARGETS darktable-cli DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT DTApplication)
diff -ur darktable-2.6.0/src/cltest/CMakeLists.txt darktable-2.6.0-rpath/src/cltest/CMakeLists.txt
--- darktable-2.6.0/src/cltest/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/cltest/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -17,7 +17,7 @@
     endif()
     set_target_properties(darktable-cltest
                           PROPERTIES
-                          INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                          INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
 endif(WIN32)
 
 install(TARGETS darktable-cltest DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT DTApplication)
diff -ur darktable-2.6.0/src/CMakeLists.txt darktable-2.6.0-rpath/src/CMakeLists.txt
--- darktable-2.6.0/src/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -857,7 +857,7 @@
     endif()
     set_target_properties(darktable
                           PROPERTIES
-                          INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                          INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
 endif(WIN32)
 
 install(TARGETS darktable DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT DTApplication)
diff -ur darktable-2.6.0/src/generate-cache/CMakeLists.txt darktable-2.6.0-rpath/src/generate-cache/CMakeLists.txt
--- darktable-2.6.0/src/generate-cache/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/generate-cache/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -17,7 +17,7 @@
     endif()
     set_target_properties(darktable-generate-cache
                           PROPERTIES
-                          INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                          INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
 endif(WIN32)
 
 install(TARGETS darktable-generate-cache DESTINATION ${CMAKE_INSTALL_BINDIR} COMPONENT DTApplication)
diff -ur darktable-2.6.0/src/imageio/format/CMakeLists.txt darktable-2.6.0-rpath/src/imageio/format/CMakeLists.txt
--- darktable-2.6.0/src/imageio/format/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/imageio/format/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -45,7 +45,7 @@
         endif()
         set_target_properties(${module}
                               PROPERTIES
-                              INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                              INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
     endif(WIN32)
     install(TARGETS  ${module} DESTINATION ${CMAKE_INSTALL_LIBDIR}/darktable/plugins/imageio/format COMPONENT DTApplication)
 endforeach(module)
diff -ur darktable-2.6.0/src/imageio/storage/CMakeLists.txt darktable-2.6.0-rpath/src/imageio/storage/CMakeLists.txt
--- darktable-2.6.0/src/imageio/storage/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/imageio/storage/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -34,7 +34,7 @@
         endif()
         set_target_properties(${module}
                               PROPERTIES
-                              INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                              INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
     endif(WIN32)
     install(TARGETS  ${module} DESTINATION ${CMAKE_INSTALL_LIBDIR}/darktable/plugins/imageio/storage COMPONENT DTApplication)
 endforeach(module)
diff -ur darktable-2.6.0/src/iop/CMakeLists.txt darktable-2.6.0-rpath/src/iop/CMakeLists.txt
--- darktable-2.6.0/src/iop/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/iop/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -51,7 +51,7 @@
       endif()
       set_target_properties(${_lib}
                             PROPERTIES
-                            INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                            INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
   endif(WIN32)
 
   install(TARGETS  ${_lib} DESTINATION ${CMAKE_INSTALL_LIBDIR}/darktable/plugins COMPONENT DTApplication)
diff -ur darktable-2.6.0/src/libs/CMakeLists.txt darktable-2.6.0-rpath/src/libs/CMakeLists.txt
--- darktable-2.6.0/src/libs/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/libs/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -128,7 +128,7 @@
         endif()
         set_target_properties(${module}
                               PROPERTIES
-                              INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                              INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
         endif(WIN32)
     install(TARGETS  ${module} DESTINATION ${CMAKE_INSTALL_LIBDIR}/darktable/plugins/lighttable COMPONENT DTApplication)
 endforeach(module)
diff -ur darktable-2.6.0/src/views/CMakeLists.txt darktable-2.6.0-rpath/src/views/CMakeLists.txt
--- darktable-2.6.0/src/views/CMakeLists.txt	2018-12-24 10:31:27.000000000 +0100
+++ darktable-2.6.0-rpath/src/views/CMakeLists.txt	2019-02-12 22:47:11.354692008 +0100
@@ -45,7 +45,7 @@
         endif()
         set_target_properties(${module}
                               PROPERTIES
-                              INSTALL_RPATH ${RPATH_DT}/../${CMAKE_INSTALL_LIBDIR}/darktable)
+                              INSTALL_RPATH ${CMAKE_INSTALL_LIBDIR}/darktable)
     endif(WIN32)
 
     install(TARGETS  ${module} DESTINATION ${CMAKE_INSTALL_LIBDIR}/darktable/views COMPONENT DTApplication)
